BitwardenShared/UI/Tools/Generator/Generator/GeneratorView.swift

```diff
@@ -48,19 +48,21 @@ struct GeneratorView: View {
             send: GeneratorAction.toastShown
         ))
         .toolbar {
-            ToolbarItemGroup(placement: .topBarTrailing) {
-                optionsToolbarMenu {
-                    Button(Localizations.passwordHistory) {
-                        store.send(.showPasswordHistory)
-                    }
-                }
-
+            ToolbarItem(placement: .topBarLeading) {
                 if store.state.presentationMode.isDismissButtonVisible {
```

Comment:
🤔 should creating the `ToolbarItem` go below the `if` condition so its only created if the presentation mode allows a dismiss button?

Reply:
SwiftUI doesn't like putting the `if` higher, as "'buildIf' is only available in application extensions for iOS 16.0 or newer".

Reply:
you're right, silly me.  My last project was iOS 17 based so I was a bit spoiled in this sense.  thanks for the reminder!